modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java

```diff
@@ -53,98 +54,117 @@
 /**
  * This class used to add WSDL files in to the governance registry using resource-admin command.
  */
-public class WSDLValidationTestCase extends GREGIntegrationBaseTest{
-    //private static final Log log = LogFactory.getLog(WSDLValidationTestCase.class);
+public class WSDLValidationTestCase extends GREGIntegrationBaseTest {
 
     private PropertiesAdminServiceClient propertiesAdminServiceClient;
-    private RelationAdminServiceClient relationAdminServiceClient;
     private ResourceAdminServiceClient resourceAdminServiceClient;
     private ServiceManager serviceManager;
     private WsdlManager wsdlManager;
-    private String sessionCookie;
     private Registry governance;
+    private final String wsdlPath = "/_system/governance/trunk/wsdls/net/restfulwebservices/www/servicecontracts"
+            + "/_2008/_01/1.0.0/GeoIPService.svc.wsdl";
 
-
-    @BeforeClass(groups = {"wso2.greg"})
+    /**
+     * THis method used to init the wsdl validation test cases.
+     *
+     * @throws Exception
+     */
+    @BeforeClass(groups = { "wso2.greg" })
     public void init() throws Exception {
         super.init(TestUserMode.SUPER_TENANT_ADMIN);
-        sessionCookie = new LoginLogoutClient(automationContext).login();
-        propertiesAdminServiceClient =
-                new PropertiesAdminServiceClient(backendURL,
-                                                 sessionCookie);
-        relationAdminServiceClient =
-                new RelationAdminServiceClient(backendURL,
-                                               sessionCookie);
-        resourceAdminServiceClient =
-                new ResourceAdminServiceClient(backendURL,
-                                               sessionCookie);
-        WSRegistryServiceClient wsRegistry =
-                new RegistryProviderUtil().getWSRegistry(automationContext);
-
-        governance = new RegistryProviderUtil().getGovernanceRegistry(wsRegistry, automationContext);
+        String sessionCookie = new LoginLogoutClient(automationContext).login();
+
+        propertiesAdminServiceClient = new PropertiesAdminServiceClient(backendURL, sessionCookie);
+        resourceAdminServiceClient = new ResourceAdminServiceClient(backendURL, sessionCookie);
+        WSRegistryServiceClient wsRegistryServiceClient = new RegistryProviderUtil().getWSRegistry(automationContext);
+
+        governance = new RegistryProviderUtil().getGovernanceRegistry(wsRegistryServiceClient, automationContext);
```

Comment:
May be it's good to change the variable name "governance" to something meaningful as the current name doesn't make much sense as to what it is used for.